### ISSUE TITLE
Bug: code inside of %%= ... =%% wrapped after 80 chars

### ISSUE THREAD

**Issue by JoernBerkefeld** [FiB3/beautyAmpCore2#4]
so here is the thing: I am running your beautyAmp extension in vscode and somehow, whenever I hit save, it autoformats everything as expected, however, it unfortunately adds linebreaks to anything inside of ampscript-print-tags:

```bash
%%=ContentBlockByName("Content Builder\Release X - BUILD\Content Blocks Library\NEW MyCompany\03. Header and banner\Referred content block\XYZ_ENV_Header_red")=%%
```
... becomes ...

```bash
%%=ContentBlockByName("Content Builder\Release X -
BUILD\Content Blocks Library\NEW MyCompany\03. Header and banner\Referred
content block\XYZ_ENV_Header_red")=%%
```

if I write the same code between %%[ ... ]%% tags then the issues does not occur, leading me to believe that %%==%% is treated as HTML content and then formatted as such.

Knowing that u use prettier internally (btw, your prettier version is outdated), I played with the config file, but unfortunately, no luck there either. The following was my final attempt at giving it ridiculously high values but without any impact on this. 

```json
{
    "useTabs": false,
    "tabWidth": 4,
    "printWidth": 500,
    "singleQuote": true,
    "trailingComma": "none",
    "overrides": [
        {
            "files": "*.html",
            "options": {
                "printWidth": 500
            }
        },
        {
            "files": "*.amp",
            "options": {
                "printWidth": 500
            }
        }
    ]
}

```

I tried finding "80" in your code, or "line" or "width" but no luck. Any thoughts from you?

**Comment by JoernBerkefeld** [comment]
looking into your code further, i think it's simply not looking at %%==%% at all? might that be the issue?
ideally, whatever is inside should be treated as ampscript and not get formatted by prettier

**Comment by FiB3** [comment]
Hello,
Ok, I can confirm the issue.
Yes, probably it's caused by Prettier, as the formatting of inline AMPscript is limited (on purpuse, as it's still "in-line" ampscript). I think I have an idea how to prevent this from happening.

Yes, I am still using the Prettier v2.X as the newer versions did not play nicely with unit tests. Will take a look if it's possible to easily update once I get to this bug.

Thanks for spotting it!

**Comment by JoernBerkefeld** [comment]
hint on prettier v2 to v3 upgrade: this is what i had to do in mcdev:

![image](https://github.com/FiB3/beautyAmpCore2/assets/1917227/d3e3a857-e293-43a4-a44e-74b328fdae61)

**Comment by JoernBerkefeld** [comment]
any news on this one? kind of a biggy TBH

**Comment by FiB3** [comment]
Not yet. As a side-side project it takes time.

**Comment by JoernBerkefeld** [comment]
all good. could you maybe give me hint on where to look? maybe i can provide a PR for it with that initial help

**Comment by JoernBerkefeld** [comment]
any news?

**Comment by JoernBerkefeld** [comment]
this is really annoying and forces us to throw out auto-formatting completely at the moment :(

**Comment by FiB3** [comment]
I want to get back to it soon. As soon as I am done with couple more pressing things.

**Comment by JoernBerkefeld** [comment]
Reminder ;-)

**Comment by FiB3** [comment]
Tons of stuff to do, but I've managed to find time.
Try: `0.4.8-beta.1` release. The Output/Print tags shouldn't get formatted.

Also included support for templating tags - which should now work in HTML/JS. You can set the tags using: `setup.ampscript.mustacheTags['{{', '}}']`.

Let me know, if everything works for you. Will release to stable afterwards.

**Comment by JoernBerkefeld** [comment]
Cool! On mustache: to avoid unwanted html encoding by mustache I advice to use triple {} instead of double

**Comment by FiB3** [comment]
Great! Let me know.
On the Mustache - you can set it as you need per my previous comment.

**Comment by FiB3** [comment]
The new `v0.4.9` contains also some minor refactors.

**Comment by JoernBerkefeld** [comment]
> Great! Let me know. On the Mustache - you can set it as you need per my previous comment.

ah, right. misunderstood you :)

**Comment by JoernBerkefeld** [comment]
hm, your new 0.4.9 is only published to npm minified? that makes debugging that much harder. would recommend you change that again. No intellisense nor ctrl-click options anymore.
opened a bug for it:
- #6

**Comment by JoernBerkefeld** [comment]
it also took away any typing / jsdocs you had in the code before, making it harder to even use it

**Comment by JoernBerkefeld** [comment]
minification/obfuscation makes sense if you want to use it in a website but in node that really only has drawbacks TBH

**Comment by JoernBerkefeld** [comment]
> Tons of stuff to do, but I've managed to find time. Try: `0.4.8-beta.1` release. The Output/Print tags shouldn't get formatted.
> 
> Let me know, if everything works for you. Will release to stable afterwards.

@FiB3 I tested 0.4.9 and it your patch did indeed work for the print tags!

**Comment by JoernBerkefeld** [comment]
found 2 improvements other than the ampscript print tags that are cool:

**80 chars line break:**
![image](https://github.com/FiB3/beautyAmpCore2/assets/1917227/8b22011a-b104-4eba-baed-7f7e57f05038)
before, it seems that line breaks in HTML were occurring after 65 chars max which was a bit weird and not what I set up in my config

**closing script tags & SSJS code properly indented:**
![image](https://github.com/FiB3/beautyAmpCore2/assets/1917227/357276a8-ea45-4340-92df-c9a1ac6ad35f)
that looks much better than before! However, somehow it also changed how style tags were rendered. I didn't update prettier or any other dependency means that must be down to something inside of your package I suppose. 

I tried looking into the why but due to #6 it's become hard to do that
